src/main/java/net/minecraftforge/network/NetworkInstance.java

```diff
@@ -28,7 +28,9 @@
  */
 @ApiStatus.Internal
 public class NetworkInstance {
```

Comment:
```suggestion
public final class NetworkInstance {
```
Internal class we don't extend ourselves, to make final to prevent unsupported extending of it